printer_mods/Mckurt/Switchwire Y Belt Tensioner/Readme.md

```diff
@@ -0,0 +1,24 @@
+| Creator | Config title | Description | V0 | V1.8 | V2.4 | VSW |
+| MCKURT11 | Switchwire Y Belt Tensioner | :x: | :x: | :x: | :x: | :heavy_check_mark: |
+
+![Screenshot_41](https://user-images.githubusercontent.com/65568463/132855469-595db6d7-ff85-4a1b-8bbc-395d22cf8505.png)
```

Comment:
Please upload your images to the `Images` directory you already have, instead of dragging and dropping in Github.  You can then include your images in the README like

    ![parts](Images/1.jpg)